components/policy-mgt/org.wso2.carbon.policy.mgt.core/src/main/java/org/wso2/carbon/policy/mgt/core/dao/impl/PolicyDAOImpl.java

```diff
@@ -1343,4 +1343,32 @@ private int readHighestPriorityOfPolicies() throws PolicyManagerDAOException {
         return priority;
     }
 
+    @Override
+    public int getPolicyCount() throws PolicyManagerDAOException {
+
```

Comment:
Removed in commit [1]
[1] https://github.com/wso2/carbon-device-mgt/commit/d7e363cef74f7700585f893253f5b0380fa26e40